site/docs/api/methods.md

```diff
@@ -381,8 +381,8 @@ The calling method syntax: `$('#table').bootstrapTable('method', parameter)`.
 
   Remove data from table, the params contain two properties:
 
-  * `field`: the field name of remove rows.
-  * `values`: the array of values for rows which should be removed.
+  * `field`: the field name of remove rows. If `$index` does not in your fields, you can use this special field `$index` to remove rows by row index.
```

Comment:
Hi @wenzhixin I think the correct statement should be something like 

If `$index` **is not** in your fields, you can use this special field `$index` to remove rows by row index.